src/main/java/com/jfrog/ide/idea/ui/configuration/XrayGlobalConfiguration.form

```diff
@@ -83,7 +83,9 @@
       </component>
       <component id="8234b" class="javax.swing.JLabel" binding="connectionResults">
         <constraints>
-          <grid row="6" column="1" row-span="1" col-span="2" vsize-policy="0" hsize-policy="0" anchor="8" fill="0" indent="0" use-parent-layout="false"/>
+          <grid row="6" column="1" row-span="1" col-span="2" vsize-policy="0" hsize-policy="0" anchor="8" fill="0" indent="0" use-parent-layout="false">
```

Comment:
```suggestion
          <grid row="6" column="1" row-span="1" col-span="2" vsize-policy="0" hsize-policy="3" anchor="8" fill="0" indent="0" use-parent-layout="false">
```
I noticed that when this cell resizes due to a long message, it changes the whole window's size the right-hand part of the configurations window gets trimmed.
I suggest changing the 'hsize-policy' to better deal with long messages.

Reply:
Right!